### PR TITLE
Add proper stat for thp on redhat

### PR DESCRIPTION
**Pull request by harish-24**
thp_split_page is not available on redhat, support has been added in this patch with thp_split

Signed-off-by: Harish <harish@linux.vnet.ibm.com>

### CODE REVIEW

File: memory/transparent_hugepages.py [hunk 1]
```diff
@@ -48,6 +49,11 @@ def setUp(self):
         self.mem_path = os.path.join(data_dir.get_tmp_dir(), 'thp_space')
         free_mem = int(memory.freememtotal() / 1024)
         self.dd_timeout = 900
+        self.thp_split = None
+        if distro.detect().name == "rhel":
+            self.thp_split = "thp_split"
+        else:
+            self.thp_split = "thp_split_page"
```
Comment (nasastry) [inline memory/transparent_hugepages.py]: The above can also be implemented in a simple way.
```
self.thp_split = "thp_split_page"
if distro.detect().name == "rhel":
    self.thp_split = "thp_split"
```

Comment (harish-24) [reply memory/transparent_hugepages.py]: Sure

**Comment by harish-24** [comment]
@nasastry Done. Thanks.

File: memory/transparent_hugepages.py [hunk 2]
```diff
@@ -48,6 +49,9 @@ def setUp(self):
         self.mem_path = os.path.join(data_dir.get_tmp_dir(), 'thp_space')
         free_mem = int(memory.freememtotal() / 1024)
         self.dd_timeout = 900
+        self.thp_split = "thp_split_page"
+        if distro.detect().name == "rhel":
```
Comment (PraveenPenguin) [inline memory/transparent_hugepages.py]: pegas release still having `thp_split_page`   we need to consider that as well

**Comment by narasimhan-v** [comment]
@harish-24 cosmetic, but a worthy change: please change redhat to rhel in commit message

**Comment by harish-24** [comment]
@narasimhan-v Sure, changed accordingly.

File: memory/transparent_hugepages.py (hunk 2 above)
Comment (narasimhan-v) [reply memory/transparent_hugepages.py]: Instead of handling based on distros, handling the IndexError with either values would be my suggestion.

Comment (harish-24) [reply memory/transparent_hugepages.py]: @narasimhan-v As some rhel releases may have `thp_split_page`, it makes sense to do it this way, will make the changes needed.

**Comment by harish-24** [comment]
@narasimhan-v @PraveenPenguin Changed based on the comments, please verify. Thanks.

**Pull request merged**